cpp/ql/test/library-tests/ir/ir/raw_consistency.expected

```diff
@@ -17,6 +17,7 @@ containsLoopOfForwardEdges
 lostReachability
 backEdgeCountMismatch
 useNotDominatedByDefinition
+| ir.cpp:1486:8:1486:8 | Unary | Operand 'Unary' is not dominated by its definition in function '$@'. | ir.cpp:1486:8:1486:8 | void StructuredBindingDataMemberStruct::StructuredBindingDataMemberStruct() | void StructuredBindingDataMemberStruct::StructuredBindingDataMemberStruct() |
```

Comment:
This seems to happen irrespective of my changes and is related to the `StructuredBindingDataMemberMemberStruct m` member I introduced in the tests.